scripts/cluster_k8s.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Probably don't have to commit this file

Reply:
Removed it and pointed it at the wiki for how to run something in the cluster